ui/admin/app/routes/scopes/scope/targets/target/index.js

```diff
@@ -29,8 +29,20 @@ export default class ScopesScopeTargetsTargetIndexRoute extends Route {
         'storage-bucket',
         storage_bucket_id,
       );
-      const storage_bucket_name = storageBucket.displayName;
-      controller.set('storage_bucket_name', storage_bucket_name);
+      const {
+        displayName: name,
+        plugin: { name: plugin_name },
+        isAWS,
+        isMinIO,
+        isUnknown,
+      } = storageBucket;
+      let icon;
+      if (isAWS) {
+        icon = 'aws';
+      } else if (isMinIO) {
+        icon = 'cloud-upload';
+      }
+      controller.set('storage_bucket', { name, plugin_name, icon, isUnknown });
```

Comment:
I didn't see this before but how come we use snake casing for some of these variables?

Reply:
yeah, I'm not sure which one to use, we do have a mix of styles in some places. But majority is camel case, I can change it to camelcase

Reply:
do you have any thoughts on this?

Reply:
I would've assumed camel case for any of our variables, since I feel like snake cases really only come from the API or modeling data.

Reply:
ty! updated to camel case